packages/dts-plugin/src/core/lib/typeScriptCompiler.ts

```diff
@@ -86,16 +89,23 @@ const createHost = (
     );
 
     for (const sourceFile of sourceFiles || []) {
-      const sourceEntry = mapExposeToEntry[sourceFile.fileName];
+      const sourceEntry = mapExposeToEntry[normalize(sourceFile.fileName)];
       if (sourceEntry) {
         const mfeTypeEntry = join(
           mfTypePath,
           `${sourceEntry}${DEFINITION_FILE_EXTENSION}`,
         );
         const mfeTypeEntryDirectory = dirname(mfeTypeEntry);
-        const relativePathToOutput = relative(mfeTypeEntryDirectory, filepath)
+        let relativePathToOutput = relative(mfeTypeEntryDirectory, filepath)
           .replace(DEFINITION_FILE_EXTENSION, '')
           .replace(STARTS_WITH_SLASH, '');
+
+        // If we're on Windows, need to convert "\" to "/" in the import path since it
+        // was derived from platform-specific file system path.
+        if (sep === '\\') {
+          relativePathToOutput = relativePathToOutput.split(sep).join('/');
+        }
```

Comment:
```suggestion
        const relativePathToOutput = relative(mfeTypeEntryDirectory, filepath)
          .replace(DEFINITION_FILE_EXTENSION, '')
          .replace(STARTS_WITH_SLASH, '')
          .replace(sep, '/');
```

Reply:
can we use `upath` here? this is how we fixed windows issues elsewhere in universe

Reply:
> can we use `upath` here? this is how we fixed windows issues elsewhere in universe

Why don't just use `path.posix` then?

I think it's good idea but I have no idea how to rewrite these files and don't break correct behavior 👉👈

It would be nice if someone could help me with this.

Reply:
I would avoid using `upath`, since the last time it has been updates is > 3 years ago

Reply:
gotcha